pkg/config/centralconfig.go

```diff
@@ -206,6 +206,7 @@ func NewCentralConfig(agentType AgentType) CentralConfig {
 		PollInterval:              60 * time.Second,
 		ClientTimeout:             60 * time.Second,
 		PlatformURL:               "https://platform.axway.com",
+		SingleURL:                 "https://ingestion.platform.axway.com",
```

Comment:
Do we want to make this default here? Or default to the URL using env overrides generated by CLI